plugins/UsersManager/Sql/UserTableFilter.php

```diff
@@ -107,7 +107,7 @@ private function getAccessSelectSqlCondition()
                 $sql = "u.superuser_access = 1";
                 break;
             default:
-                $sql = "a.access = ?";
+                $sql = "u.login IN (SELECT login from " . Common::prefixTable('access') . " WHERE access = ?)";
```

Comment:
This change is needed, as the resulting query tries to do a `GROUP_CONCAT` on `a.access`, so we are able to fetch all available access types. Not using a subquery here results in returning only the specific access type it was filtered for.